.github/workflows/post-pr.yml

```diff
@@ -31,7 +31,7 @@ jobs:
   ui-tests:
     name: UI Tests (Synapse)
     needs: should-i-run
-    runs-on: macos-latest
+    runs-on: buildjet-4vcpu-ubuntu-2204
```

Comment:
We start with this runner and will compare later the perf of other available (but more not with the same price) runners.

Reply:
Can we try first with  `buildjet-2vcpu-ubuntu-2204`?
And use beefier machines only if we can prove there are better?

Reply:
Yes. 0685fb1e1a16cb2dce12ed2e2785db31e8efab46